APMrover2/commands_logic.pde

```diff
@@ -330,8 +330,11 @@ static void do_take_picture()
 // log_picture - log picture taken and send feedback to GCS
 static void log_picture()
 {
-    gcs_send_message(MSG_CAMERA_FEEDBACK);
-    if (should_log(MASK_LOG_CAMERA)) {
-        DataFlash.Log_Write_Camera(ahrs, gps, current_loc);
+    if(camera._feedback_pin == -1 ){
```

Comment:
This will break current functionality, if the user does not have the feedback pin connected this condition will never be true, I think we should add a parameter to enable or diasable this precise logging option.